tests/snapshots/assets__check_file@unit-grid-basic-cmt.typ-120.snap

```diff
@@ -4,13 +4,19 @@ expression: doc_string
 input_file: tests/assets/unit/grid/basic-cmt.typ
 snapshot_kind: text
 ---
+#table(columns: 2, [Name], [Age], [Hannes], [10], /* 1 */)
 #table(
-  columns: 2,
-  [Name], [Age],
-  [Hannes], [36],
+  columns: 2, [Name], [Age], [Hannes], [20], /* 2 */
 )
-#table( columns: 2,[Name], [Age], /*  */ [Hannes], [36],/*  */
-
+#table(
+  columns: 2, [Name], [Age], [Hannes], [30], /* 3 */
+)
+#table(
+  columns: 2, [Name], [Age], /* 4 */ [Hannes], [40], /* 5 */
+)
+#table(columns: 2, [Name], [Age], /* 6 */ [Hannes], /* 7 */ [50],)
+#table(
+  columns: 2, [Name], /* 8 */ [Age], [Hannes], [60], // wwww
 )
```

Comment:
they look very different after the change. Is this expected?

Reply:
I modified this case. Everything meets expectations.